circuit_maintenance_parser/parser.py

```diff
@@ -55,32 +70,19 @@ class ICal(Parser):
 
     _data_types = ["text/calendar", "ical", "icalendar"]
 
-    def parse(self, raw: bytes) -> List[Dict]:
-        """Method that returns a list of Maintenance objects."""
-        result = []
-
+    def parser_hook(self, raw: bytes):
```

Comment:
Can we document somewhere the conventions for the various `parse` methods and how they interrelate? We now have `parse`, which calls `parser_hook`, which can either call `parse_ical`, `parse_html` (which in turn might call `parse_body`, `parse_bs`, `parse_div`, `parse_p`, `parse_tables`, and/or `parse_title`), `parsedate_tz`, `parse_subject`, or `parse_csv` (is that all of them?)

Reply:
Will do!